brave-asynchttpclient/brave-asynchttpclient-2.x/src/main/java/smartthings/brave/asynchttpclient/ClientTracing.java

```diff
@@ -61,7 +61,7 @@ private ClientTracing() {}
 
   public static final class TracingRequestFilter extends TracingFilter implements RequestFilter {
 
-    TracingRequestFilter(HttpTracing httpTracing) {
+    public TracingRequestFilter(HttpTracing httpTracing) {
```

Comment:
generally a bit nicer to expose a public factory method vs constructor as it lets us do other things later. otoh I don't have anything in mind that would be limited by this (plus the type is final)